### ISSUE TITLE
Misleading documentation/usage of "druid.indexing.doubleStorage" property

### ISSUE THREAD

**Issue by sascha-coenen** [apache/incubator-druid#8677]
### Affected Version

0.11.x and following

### Description
The release notes of Druid 0.11.0 (https://github.com/apache/incubator-druid/releases/tag/druid-0.11.0) state that the "druid.indexing.doubleStorage" property could be specified in the common.runtime.properties file:
"To enable Double column storage, set the following property in the common runtime properties"
Furthermore, the Druid codebase contains example common.runtime.properties files in which this property is set:
https://github.com/apache/incubator-druid/blob/druid-0.16.0-incubating/examples/conf/druid/cluster/_common/common.runtime.properties

However, both the Druid source code and also the Druid documentation seem to suggest that this property needs to be set as a java system property:
in class ColumnHolder
`
static boolean storeDoubleAsFloat()
{
  String value = System.getProperty(DOUBLE_STORAGE_TYPE_PROPERTY, "double");
  return !"double".equals(StringUtils.toLowerCase(value));
}
`

Likewise the documentation correctly states:
https://github.com/apache/incubator-druid/blob/druid-0.16.0-incubating/docs/configuration/index.md

"To keep the old format set the system-wide property druid.indexing.doubleStorage=float"

In summary, this property only comes into effect if set as a java system property. Setting it within the commons.runtime.properties file has no effect.
I propose to either remove the property from the common.runtime.properties example files and to correct the release notes OR to add support for specifying this property in the configuration files. I would prefer the later option.
